packages/open-next/src/core/patchAsyncStorage.ts

```diff
@@ -1,9 +1,9 @@
-const mod = require("node:module");
+let mod = require("node:module");
 
 const resolveFilename = mod._resolveFilename;
 
 export function patchAsyncStorage() {
-  mod._resolveFilename = ((
+  const _resolveFilename = ((
```

Comment:
I don't think this is correct path on workers.

`_resolveFilename` is not used on workers ([it throws](https://github.com/unjs/unenv/blob/b18778548152d2a9a019b8cb42a3367508d8960d/src/runtime/node/module.ts#L127-L129)) as we do not use dynamic require - requires/imports are resolved at build time.

Reply:
Maybe we can use a build time patch instead, replacing the Next version with the patch

Reply:
ah ok I see, yes that's a good call thanks 🙂👍 

I'm happy to close the PR if we don't see the point in this then (however it is a bit too bad that aws does something that then we need to un-do, I wonder if that could be solved somehow at some point 😕)